app/models/tenant_quota.rb

```diff
@@ -49,10 +49,41 @@ class TenantQuota < ApplicationRecord
   scope :vms_allocated,       -> { where(:name => :vms_allocated) }
   scope :templates_allocated, -> { where(:name => :templates_allocated) }
 
+  virtual_column :name_formatted, :type => :string
+  virtual_column :total_formatted, :type => :integer
+  virtual_column :used_formatted, :type => :float
+  virtual_column :allocated_formatted, :type => :float
+  virtual_column :available_formatted, :type => :float
+
   before_validation(:on => :create) do
     self.unit = default_unit unless unit.present?
   end
 
+  def quota_value_formatted(quota_value)
```

Comment:
In not sure that we need to add this method for formatting the quota values. Shouldn't the built-in formats that you get on the "Formatting" tab in the report editor do this?